config/webpack/docs/webpack.config.dev.js

```diff
@@ -33,15 +33,19 @@ module.exports = {
   },
 
   resolve: {
-    extensions: ["", ".js", ".jsx"]
+    extensions: ["", ".js", ".jsx", ".json"]
   },
 
   module: {
     loaders: [
       {
         test: /\.jsx?$/,
         exclude: [/node_modules/],
-        loaders: [require.resolve("babel-loader")]
+        loaders: [require("builder-victory-component-dev/require").resolve("babel-loader")]
```

Comment:
This one is not correct though because the `babel-loader` dep is in prod archetype: https://github.com/FormidableLabs/builder-victory-component/blob/master/package.json#L61